app/views/admin/consultation/_index.html.haml

```diff
@@ -0,0 +1,3 @@
+%ul
```

Comment:
This file is generated the following rake task:
`bundle exec rake generate_active_admin_resources[consultation]`